contracts/ERC20.sol

```diff
@@ -0,0 +1,364 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts v4.4.1 (token/ERC20/ERC20.sol)
+
+pragma solidity ^0.8.0;
+
+import "../openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
```

Comment:
You can do `npm install @openzeppelin/contracts` and then import ERC20 contracts like `import "@openzeppelin/contracts/token/ERC20/ERC20.sol";`